db/migrate/20201105122835_add_company_name_to_billing_contracts.rb

```diff
@@ -0,0 +1,5 @@
+class AddCompanyNameToBillingContracts < ActiveRecord::Migration[6.0]
+  def change
+    add_column :billing_contracts, :company_name, :string
```

Comment:
We have table companies. You mustn't add column company_name for billing_contracts table!